### ISSUE TITLE
When I change the batch_size to a number more than 1, it comes to the error of dimension not match

### ISSUE THREAD

**Issue by WenqingHua** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#80]
The default batch_size is 1 and I can't understand for it makes the loss waving.
![image](https://user-images.githubusercontent.com/28468626/42016923-d539458a-7adf-11e8-9d06-9afefcc284e2.png)
So I change it to 2 and so on.
![image](https://user-images.githubusercontent.com/28468626/42016957-f19405d0-7adf-11e8-9390-774e923df106.png)
But it come to the error as follows when I training as normal.
![image](https://user-images.githubusercontent.com/28468626/42017005-287924ae-7ae0-11e8-9416-3a1aaf506038.png)
![image](https://user-images.githubusercontent.com/28468626/42017033-3d9b8764-7ae0-11e8-837e-f3d668080909.png)

**Comment by leccyril** [comment]
i read we can not configurer batch_size to 1

**Comment by WenqingHua** [comment]
@leccyril Where

**Comment by leccyril** [comment]
don't remember but maybe you pictures have not same size ? it is recommanded to let one... it is the default configuration.. maybe try to ask directlty to tensorflow github ?
